python-opendata/requirements.txt

```diff
@@ -0,0 +1,5 @@
+arrow==0.12.1
```

Comment:
I would avoid to pin the version if you can.

Reply:
Completely agree. I actually used the Pipfile for development but since Pipenv is not very suited for library development I want to move to a requirements.txt. I just did a simple `pip freeze > requirements.txt`